tests/test_cmab.py

```diff
@@ -311,6 +316,33 @@ def run_predict(mab):
     assert mab != create_cmab_bernoulli_cold_start(action_ids=["a1", "a2", "a3", "a4", "a5"], n_features=n_features)
     run_predict(mab=mab)
 
+    @pytest.mark.parametrize("action_dict, action_ids", [
```

Comment:
You can use hypothesis as in all the other tests. It allows you to try many more combinations with much less and clean code.